frontend/src/services/annotator/spectro/spectro.context.tsx

```diff
@@ -29,6 +34,9 @@ export interface SpectroCtx {
   currentParams?: SpectrogramParams;
   availableParams: Array<SpectrogramParams>
 
+  currentColormap: SpectrogramColormap;
+  availableColormaps: Array<string>;
```

Comment:
Does this need to be in a context ? Isn't it a const that can be used anywhere ?